test/components/app-test.js

```diff
@@ -0,0 +1,19 @@
+const assert = require('assert')
+
+const App = require('../../components/app/app')
+const jsdom = require('jsdom')
+const React = require('react')
+const ReactDOM = require('react-dom')
+const TestUtils = require('react-addons-test-utils')
+
+describe('App', function() {
+  before(function() {
+    global.document = jsdom.jsdom('<!doctype html><html><body></body></html>')
+    global.window = global.document.defaultView
```

Comment:
It's a node thing https://nodejs.org/api/globals.html#globals_global. It's basically the `window` var of node.